src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -120,7 +120,8 @@ @implementation MTRDeviceController {
     MTRDeviceAttestationDelegateBridge * _deviceAttestationDelegateBridge;
     MTRDeviceControllerFactory * _factory;
     NSMapTable * _nodeIDToDeviceMap;
-    os_unfair_lock _deviceMapLock; // protects nodeIDToDeviceMap
+    os_unfair_lock_t _deviceMapLock; // protects nodeIDToDeviceMap
```

Comment:
This line is not needed because there is already an underlying lock. The property is just a convenience to access the address.

Reply:
great catch - removed.